src/pushsource/_impl/model/erratum.py

```diff
@@ -368,12 +376,20 @@ def _from_data(cls, data):
         if data.get("cdn_repo"):
             kwargs["dest"] = data["cdn_repo"]
 
+        # If source of data is ET/staging directory, try to get container_list directly
+        if data.get("container_list"):
+            kwargs["container_list"] = data["container_list"]
```

Comment:
This doesn't seem right, because the  model is a flat list of strings, but this is not what ET returns.

An example of what ET returns can be found at https://url.corp.redhat.com/26b7928

Reply:
fixed